docs/android-core-new/participants/events.mdx

```diff
@@ -1,192 +1,230 @@
+            // you can use `meeting.participants.screenshares` to get latest screenshare participants
+        }
 
-Trigger an event when any participant leaves the meeting.
+        override fun onScreenShareStarted(participant: DyteJoinedMeetingParticipant) {
+            // participant stared presenting screen in the meeting
+        }
 
-```ts
-meeting.participants.joined.on('participantLeft', (participant) => {
-  console.log(`A participant with id "${participant.id}" has left the meeting`);
-});
+        override fun onScreenShareEnded(participant: DyteJoinedMeetingParticipant) {
+            // participant stopped presenting screen in the meeting
+        }
+    }
 ```
 
-### Participant pinned
+## Active speaker
+
+Trigger an event when any is change in active speaker in the meeting.
 
-Trigger an event when a participant is pinned.
+```kotlin
+    private val participantEventsListener = object : DyteParticipantEventsListener {
+        override fun onActiveSpeakerChanged(participant: DyteMeetingParticipant) {
+            // your code here to handle active speaker
+        }
 
-```ts
-meeting.participants.joined.on('pinned', (participant) => {
-  console.log(`Participant with id "${participant.id}" was pinned`);
-});
+        override fun onNoActiveSpeaker() {
+            // your code here to handle no active speaker
+        }
+    }
 ```
 
-### Participant unpinned
+## Pinned participant
+
+Trigger an event when any is change in pinned participant in the meeting.
 
-Trigger an event when a participant is unpinned.
+```kotlin
+    private val participantEventsListener = object : DyteParticipantEventsListener {
+        override fun onParticipantPinned(participant: DyteMeetingParticipant) {
+            // your code here to show pinned participant
+        }
 
-```ts
-meeting.participants.joined.on('unpinned', (participant) => {
-  console.log(`Participant with id "${participant.id}" was unpinned`);
-});
+        override fun onParticipantUnpinned() {
+            // your code here to remove pinned participant
+        }
+    }
 ```
 
-### Video update
+## Active participants list change
 
-Trigger an event when any participant starts / stops video.
+Triggers an event when any is change in active participants list in the meeting.
 
-```ts
-meeting.participants.joined.on('videoUpdate', (participant) => {
-  console.log(
-    `A participant with id "${participant.id}" updated their video track in the meeting`
-  );
-  // Use the video track if it exists
-  if (participant.videoEnabled) {
-    // participant.videoTrack
-  } else {
-    // handle stop video
-  }
-});
+```kotlin
+    private val participantEventsListener = object : DyteParticipantEventsListener {
+        override fun onActiveParticipantsChanged(active: List<DyteMeetingParticipant>) {
+            // your code here to refresh active participants
+        }
+    }
 ```
 
-### Audio update
+## All participants list change
```

Comment:
(change-request): I think we can skip the documentation related to the `all` list. It currently applies only to Chat meetings. We will add it once we release the Chat SDK.